### PR TITLE
Fix last step in recovery-tool.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9663.

### CODE REVIEW

File: Teams/rooms/recovery-tool.md [hunk 1]
```diff
@@ -41,11 +41,10 @@ Download the latest [Microsoft Teams Rooms installation package](https://go.micr
       3. Beneath **Reset this PC**, select **Get started**
       4. Select **Remove everything**, then **Next** and **Reset**
         > [!WARNING]
```
Comment (JohanFreelancer9) [inline Teams/rooms/recovery-tool.md:43]:
```suggestion

                > [!WARNING]
```

File: Teams/rooms/recovery-tool.md [hunk 2]
```diff
@@ -41,11 +41,10 @@ Download the latest [Microsoft Teams Rooms installation package](https://go.micr
...
-        > The Microsoft Teams Rooms device can become unusable if the **Keep my files - Removes Apps and settings, but keeps your personal files** option is selected during the Windows Reset process. Do not select this option.
+        > The Microsoft Teams Rooms device can become unusable if the **Keep my files - Removes Apps and settings, but keeps your personal files** option is selected             during the Windows Reset process. Do not select this option.
```
Comment (JohanFreelancer9) [inline Teams/rooms/recovery-tool.md:44]:
```suggestion
        > The Microsoft Teams Rooms device can become unusable if the **Keep my files - Removes Apps and settings, but keeps your personal files** option is selected during the Windows Reset process. Do not select this option.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review..

File: Teams/rooms/recovery-tool.md [hunk 3]
```diff
@@ -41,11 +41,10 @@ Download the latest [Microsoft Teams Rooms installation package](https://go.micr
       3. Beneath **Reset this PC**, select **Get started**
       4. Select **Remove everything**, then **Next** and **Reset**
```
Comment (JohanFreelancer9) [inline Teams/rooms/recovery-tool.md:42]:
```suggestion
      4. Select **Remove everything**, then **Next** and **Reset**
      
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**